pkg/Library/Mediapipe/HolisticSticker.js

```diff
@@ -11,9 +11,9 @@ async initialize(_, state, {service}) {
 shouldUpdate({data}) {
   return Boolean(data);
 },
-async update({data}, state, {service}) {
+async update({data, sticker}, state, {service}) {
   await service({kind: 'MediapipeService', msg: 'clear', data: {target: state.target}});
-  await service({kind: 'MediapipeService', msg: 'renderSticker', data: {data: data.results, target: state.target}});
+  await service({kind: 'MediapipeService', msg: 'renderSticker', data: {data: data.results, sticker: sticker, target: state.target}});
```

Comment:
In this case you want to send `sticker: sticker.canvas`

If it doesn't work because `sticker.canvas` is always `null` then that's a bug for me.

Reply:
Done!